lib/Serialization/ModuleFile.cpp

```diff
@@ -375,8 +375,8 @@ ValidationInfo serialization::validateSerializedAST(
   ValidationInfo result;
 
   // Check 32-bit alignment.
-  if (data.size() % 4 != 0 ||
-      reinterpret_cast<uintptr_t>(data.data()) % 4 != 0)
+  if (data.size() % SWIFTMODULE_ALIGNMENT != 0 ||
+      reinterpret_cast<uintptr_t>(data.data()) % SWIFTMODULE_ALIGNMENT != 0)
```

Comment:
It's probably not worth changing right now, but in the future, perhaps we could use the new Alignment.h helpers, like:
```
Align moduleAlignment = SWIFTMODULE_ALIGNMENT;
if (!isAligned(moduleAlignment, data.size()) || !isAddrAligned(moduleAlignment, data.data())) ...
```

Reply:
Oh, the `// Check 32-bit alignment` comment may need an update.